src/lib/header-bidding/prebid/bid-config.ts

```diff
@@ -223,17 +224,35 @@ const openxClientSideBidder: (pageTargeting: PageTargeting) => PrebidBidder = (
 	},
 });
 
-const getOzonePlacementId = () => (isInUsa() ? '1420436308' : '0420420500');
+const getOzonePlacementId = (sizes: HeaderBiddingSize[]) => {
+	if (isInUsa()) {
+		if (getBreakpointKey() === 'D') {
+			if (containsBillboard(sizes)) {
+				return '3500010912';
+			}
+
+			if (containsMpu(sizes)) {
+				return '3500010911';
+			}
+		}
+		return '1420436308';
+	} else {
+		return '0420420500';
+	}
+};
 
 const ozoneClientSideBidder: (pageTargeting: PageTargeting) => PrebidBidder = (
 	pageTargeting: PageTargeting,
 ) => ({
 	name: 'ozone',
 	switchName: 'prebidOzone',
-	bidParams: (): PrebidOzoneParams => ({
+	bidParams: (
+		_slotId: string,
```

Comment:
Is this slotId actually referenced at any point?

Reply:
Nope, but it needs to be there to access the second argument, the _ prefix tells typescript/vscode to ignore the unused variable.